webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/services/policies/write/PolicyWriterService.scala

```diff
@@ -112,17 +111,40 @@ trait PolicyWriterService {
 }
 
 object PolicyWriterServiceImpl {
-  val defaultPermissions = Set[PosixFilePermission](PosixFilePermission.OWNER_READ, PosixFilePermission.OWNER_WRITE
-                                                  , PosixFilePermission.GROUP_READ)
-  val defaultFolderPermissions = Set[PosixFilePermission](PosixFilePermission.OWNER_READ, PosixFilePermission.OWNER_WRITE, PosixFilePermission.OWNER_EXECUTE
-                                                  , PosixFilePermission.GROUP_READ, PosixFilePermission.GROUP_EXECUTE)
+  import PosixFilePermission._
+  import StandardOpenOption._
+  // we want: /bin/chmod u-x,u+rwX,g-wx,g+rX,o-rwx ...
+  val filePerms      = Set[PosixFilePermission](OWNER_READ, OWNER_WRITE, GROUP_READ)
```

Comment:
i know it's nt introduced by this PR, but permissions for root ought to be 600 (we have policies that change it when agent is ran)
Do you think you can make a specific case for that ?